### ISSUE TITLE
Git Commit Message Conventions

### ISSUE THREAD

**Issue by dreytko** [Bledhard/GreenShop#22]
I've found an article related to the Commit Message Conventions.
In my opinion, it is pretty good and we can utilize for our improvement.

Here's the link on it: [AngularJS Git Commit Message Conventions](https://gist.github.com/stephenparish/9941e89d80e2bc58a153)

**Comment by dreytko** [comment]
> I've found an article related to the Commit Message Conventions.
> In my opinion, it is pretty good and we can utilize for our improvement.
> 
> Here's the link on it: [AngularJS Git Commit Message Conventions](https://gist.github.com/stephenparish/9941e89d80e2bc58a153)

After reviewing this doc, I've found that it is not perfect and their examples break the rules.
So I've found this doc, with seven simple rules. I will test the idea mentioned there, and, in case it works good for us, I will add these rules somewhere in the Wiki pages.
Here's [the link](https://chris.beams.io/posts/git-commit).

FYI: [another commit message guideline](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53).
